src/components/Personalization/index.js

```diff
@@ -105,24 +107,32 @@ const createPersonalization = ({ config, logger, eventManager }) => {
 
   return {
     lifecycle: {
-      onBeforeEvent({ event, isViewStart, scopes }) {
+      onBeforeEvent({ event, isViewStart, scopes = [] }) {
         if (authoringModeEnabled) {
           logger.warn("Rendering is disabled, authoring mode.");
 
           // If we are in authoring mode we disable personalization
           mergeQuery(event, { enabled: false });
           return;
         }
+        const hasScopes = scopes.length > 0;
+        const queryDetails = {};
 
         // For viewStart we try to hide the personalization containers
         if (isViewStart) {
           hideContainers(prehidingStyle);
         }
 
-        if (isViewStart || scopes) {
+        if (isViewStart || hasScopes) {
           event.expectResponse();
-          mergeQuery(event, { scopes });
+          queryDetails.accepts = allSchemas;
```

Comment:
Shouldn't this be in the `if (hasScoped)` clause?

Reply:
nope, schemas are the representations of decisions items